docs/real-user-metrics.md

```diff
@@ -0,0 +1,60 @@
+# Real User Metrics
+
+Real user metrics allow a user's browser to report on how a page loaded using the [Performance API]. The tool that we use to do this is called [LUX] - short for Live User Experience - and is run by Speedcurve.
+
+The benefit of RUM is that it shows how pages are performing in real situations, rather than on synthetic tests. This means that we don't have to guess at what conditions might be - RUM provides insight into what conditions actually are.
+
+## Loading
+
+The LUX scripts should only be loaded when a user has opted into usage tracking. This is done using the `rum-loader` script, which checks if the `usage` cookie is `true` and then loads the required scripts. This is already part of the [Public Layout component].
+
+## Custom changes when updating LUX
+
+The scripts for the real user metrics are loaded from our servers - this allows us to know what is contained within the scripts and reduce the risk of unwelcome things being added to GOV.UK
+
+Because of this, the scripts are audited before being updated and then two extra lines are added to make sure it works correctly. These two lines set the customer ID and turn on beacon mode.
+
+### Customer ID
+
+When loading `lux.js` from Speedcurve's CDN, the customer ID is appended to the end of the URI as a query string. The script looks for a script in the DOM with a source of `lux.js`, and from that extracts the customer ID.
```

Comment:
Just to clarify - this is GOV.UK's customer ID, not something that identifies a user? I think this might be worth clarifying.

Reply:
Yes, this is the ID of us as a customer of Speedcurve - not the ID of the user visiting GOV.UK. I'll amend to emphasise this.